### PR TITLE
Kluys fluff item + Pirate coat fix

### PR DESCRIPTION
**Pull request by Bram961**
Adds in the nanofibre jacket, a jacket that can switch between being
concealed, jacket and coat mode.
Fixes the black pirate coat and its tiny white strip on one of the arms
that was meant to be part of the undersuit already in the game.

### CODE REVIEW

**Comment by Bram961** [comment]
http://i.imgur.com/ik45hPz.png Picture of the change to the pirate coat (might be the other side) as it was intended to have the white strip removed after testing it with the undersuit that goes with it.
Which was done to one side but not the other.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -504,4 +504,37 @@
...
+//Kluys fluff nanofibre suit
+/obj/item/clothing/suit/kluysfluff
+	name = "Nano Fibre Jacket"
+	desc = "A Black Suit made out of nanofibre. The newest of cyberpunk fashion using hightech liquid to solid materials."
+	icon_state = "Kluysfluff1"
+	item_state = "Kluysfluff1"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|ARMS
+
+	verb/toggle()
```
Comment (TheDZD) [inline code/modules/clothing/suits/miscellaneous.dm]: Line should be `/obj/item/clothing/suit/kluysfluff/verb/toggle()` as relative pathing shouldn't be used unless the entire file is using it.
Additionally, this is in the wrong file. Fluff items should go [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/customitems/item_defines.dm).

**Comment by Bram961** [comment]
Placed in the correct location, removed from the previous location, Line changed to the stated line in the previous comment.

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -146,6 +146,38 @@
...
+//Kluys fluff nanofibre suit
+/obj/item/clothing/suit/kluysfluff
+	name = "Nano Fibre Jacket"
+	desc = "A Black Suit made out of nanofibre. The newest of cyberpunk fashion using hightech liquid to solid materials."
+	icon_state = "Kluysfluff1"
+	item_state = "Kluysfluff1"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|ARMS
+
+	/obj/item/clothing/suit/kluysfluff/verb/toggle()
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: This is technically /obj/item/clothing/suit/kluysfluff/obj/item/clothing/suit/kluysfluff/verb/toggle().

Also known as; Should be tabbed down one.

Comment (TheDZD) [inline code/modules/customitems/item_defines.dm]: Yup, just remove one tab from everything inside of the toggle verb.

**Comment by Bram961** [comment]
That should fix the last issue.

**Pull request merged**